### PR TITLE
Pwned - Added DBot Score 

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/14507

## Does it break backward compatibility?
   - No

## Must have
- [ ] Code Review

### CODE REVIEW

File: Integrations/integration-Pwned.yml [hunk 1]
```diff
@@ -76,12 +77,26 @@ script:
...
+            if (DEFAULT_DBOT_SCORE === 3){
+                email.Malicious = {
+                    Vendor: 'Pwned',
+                    Description: 'The email has been compromised',
+                }
+            }
             addCompromised(compEmail, outputPaths.email, {
                 'Address': email,
-                'Compromised': {Vendor:'Pwned', Reason: compSites.join(', ')}
+                'Compromised': {Vendor:'Pwned', Reason: compSites.join(', '), dBotScore: DEFAULT_DBOT_SCORE}
```
Comment (yaakovi) [inline Integrations/integration-Pwned.yml]: anyway, there shouldn't dbot score here 
```suggestion
                'Compromised': {Vendor:'Pwned', Reason: compSites.join(', '), DBotScore: DEFAULT_DBOT_SCORE}
```

File: Integrations/integration-Pwned.yml [hunk 2]
```diff
@@ -76,12 +77,26 @@ script:
...
             });
         }
+        var dBotScore = {
```
Comment (yaakovi) [inline Integrations/integration-Pwned.yml]: what is this?
this is unused variable...

File: Integrations/integration-Pwned.yml [hunk 3]
```diff
@@ -91,12 +106,27 @@ script:
...
+                }
+            }
+
             addCompromised(compDomain, outputPaths.domain, {
                 'Name': domain,
-                'Compromised': {Vendor:'Pwned', Reason: compSites.join(', ')}
+                'Compromised': {Vendor:'Pwned', Reason: compSites.join(', '), dBotScore: DEFAULT_DBOT_SCORE}
             });
         }
+        var dBotScore = {
```
Comment (yaakovi) [inline Integrations/integration-Pwned.yml]: unused variable

**Comment by Shellyber** [comment]
There wan to tests for this integration I think

**Pull request merged**